core/include/gnuradio-4.0/BlockTraits.hpp

```diff
@@ -391,11 +424,10 @@ concept processBulk_requires_ith_output_as_span
                                                         traits::block::stream_output_port_types<TDerived>>::template apply<std::tuple>
                            bad_outputs) {
               {
```

Comment:
could `I`  be range checked against the number of outputs to catch invalid use of the template? Not super important just noticed while playing with it that I had accidentally requested out of range indices.

Reply:
Done